### ISSUE TITLE
我还要

### ISSUE THREAD

**Issue by lucylucy27** [mkkellogg/GaussianSplats3D#362]

**Comment by lucylucy27** [comment]
Hi mkkellogg,

I found on Iphone13 with 4G GPU memory，the demo fails to load (but it happens to be successful!). 
If i change the ply model to smaller size, the demo works well. 
My question is, why the memory usage will increase sharply in loading? It may be the block reason to open bigger size models.
I've also tried upgrading to v0.4.5, but there hasn't been much improvement compared with v0.4.2
Here is my config to open *.ply file in release v0.4.5:

'gpuAcceleratedSort': true,
'antialiased': false,
'sharedMemoryForWorkers':false,
'sceneRevealMode': GaussianSplats3D.SceneRevealMode.Instant,
'freeIntermediateSplatData': true,
'plyInMemoryCompressionLevel': 0,
'optimizeSplatData': false,
'sphericalHarmonicsDegree': 2
'progressiveLoad': false

Is there any thing I'm missing?

Thanks very much in advance~
